src/state/slices/validatorDataSlice/utils.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  AccountId,
+  AssetId,
+  ChainId,
+  cosmosChainId,
+  fromAccountId,
+  fromAssetId,
+  osmosisChainId,
+} from '@shapeshiftoss/caip'
+import flow from 'lodash/flow'
+
+import {
+  SHAPESHIFT_COSMOS_VALIDATOR_ADDRESS,
+  SHAPESHIFT_OSMOSIS_VALIDATOR_ADDRESS,
+} from './constants'
+
+export const getDefaultValidatorAddressFromChainId = (chainId: ChainId) => {
+  switch (chainId) {
+    case cosmosChainId:
+      return SHAPESHIFT_COSMOS_VALIDATOR_ADDRESS
+    case osmosisChainId:
+      return SHAPESHIFT_OSMOSIS_VALIDATOR_ADDRESS
+    default:
+      return ''
```

Comment:
i'd be more comfortable throwing `unimplemented` here to fail faster. it won't change the consumption or type sig. 

little concerned about this causing a bug downstream in the future